Lib/asyncio/unix_events.py

```diff
@@ -102,7 +102,13 @@ def add_signal_handler(self, sig, callback, *args):
             # main thread.  By calling it early we ensure that an
             # event loop running in another thread cannot add a signal
             # handler.
-            signal.set_wakeup_fd(self._csock.fileno())
+            oldfd = signal.set_wakeup_fd(self._csock.fileno())
+            if oldfd != -1 and oldfd != self._csock.fileno():
+                warnings.warn(
+                    'Overriding signal wakeup file '
+                    'descriptor for loop signal handlers',
```

Comment:
```suggestion
                    "Signal wakeup fd was already set",
```

Reply:
Done in https://github.com/python/cpython/pull/96807/commits/bc7b849d3b29b643f0352441142e95fb0a72bcbe.